### ISSUE TITLE
Sot-23 3D-Rotation

### ISSUE THREAD

**Issue by Gijsco** [KiCad/TO_SOT_Packages_SMD.pretty#63]
When adding the SOT-23 or the SOT-23_Handsoldering the 3d image is rotated 90 degree from the pads.

**Comment by Shackmeister** [comment]
Please update your footprints and 3D models, that should solve the issue

**Comment by Gijsco** [comment]
Dear Frank,

Thank you for your quick reply and pointing out how to fix it.

To update the the footprints I deleted the local
"TO_SOT_Packages_SMD.pretty" from "C:\Program
Files\KiCad\share\kicad\modules" in order to be reloaded by kicad. When
going back to the library browser the 3D package is still rotated.
[image: Inline afbeelding 1]

Replacing all items in "Packages3D" solved the issue. I did not know that
the 3D packages are not updated automatically. Sorry for placing a ticket
for something that has already been solved.


Yours faithfully,
Gijs van den Eerenbeemt

2017-09-26 9:44 GMT+02:00 Frank Severinsen <notifications@github.com>:
